apps/extension/src/ui/domains/Sign/SignRequestContext/PolkadotSignRequestContext.ts

```diff
@@ -69,7 +75,7 @@ const useDryRun = ({
     queryFn: async () => {
       if (!from || !sapi || !decodedCall) return null
 
-      return sapi.getDryRunCall(from, decodedCall)
+      return sapi.getDryRunCall<DryRunResult>(from, decodedCall)
```

Comment:
typescript won't allow `@talismn/sapi` to export a function whose type signature depends on `@polkadot-api/descriptors`

as a workaround, I made this function return a type based on `T`, which we can pass in here as the `DryRunResult` we're expecting to get back

in the future we can move `@polkadot-api/descriptors` into a real package like `@talismn/registry` (kudos @0xKheops) at which point we'll be able to use its types publicly